### DISCUSSION TITLE
How to use Commands not Usable in Normal Mode?

### DISCUSSION THREAD

**Discussion by abreathingcorpse** [qutebrowser/qutebrowser#7836] [Q&A] [answered]
The [Docs](https://qutebrowser.org/doc/help/commands.html) show some commands which are not usable in Normal Mode. How is one supposed to use them? Where could I find more documentation on those commands?

**Comment by The-Compiler** [comment, marked as answer]
Typically by binding them to a key. All keybinds in all modes are just keys bound to those commands. What kind of "more documentation" is missing? There isn't really much other than the thing you linked to.

**Reply by abreathingcorpse** [reply]
TY very much! I didn't understand that's how one was supposed to run these commands.
